app/assets/javascripts/models/beta/project.js

```diff
@@ -0,0 +1,16 @@
+import * as Label from './label';
+
+export const deserialize = (board) => ({
+  ...board.project,
+  labels: Label.splitLabels(board.labels)
+});
+
+export const addLabel = (project, label) => (
+  {
```

Comment:
I think it's better if you let the `{` right on the side of the parenthesis, you can save two lines and make the legibility better.

```js
export const addLabel = ({
  ...project,
  labels: [
    ...project.labels,
    label
  ]
})
```

Reply:
fancier! :smile: